integration-test/src/main/java/org/commonjava/maven/ext/integrationtest/TestUtils.java

```diff
@@ -248,9 +250,21 @@ static Integer runMaven( String commands, Map<String, String> params, String wor
         throws Exception
     {
         String stringParams = toJavaParams( params );
-        String commandMaven = String.format( "mvn %s %s ", commands, stringParams );
+        String commandMaven;
 
-        return runCommandAndWait(commandMaven, workingDir, MVN_LOCATION + "/bin");
+        if ( SystemUtils.IS_OS_WINDOWS )
+        {
+            commandMaven = String.format( "cmd.exe /C \"%s %s %s\"", MVN_LOCATION + File.separator + "bin"
```

Comment:
Maybe it should repeat something like this below on line 262 and get rid of line 374? Overriding the `PATH` like this doesn't seem to work on Windows, as Windows doesn't exec in a shell it doesn't honor the `PATH`. I'd say it would work except that something else is being added to the `PATH`.

Reply:
@dwalluck Interestingly it used to be something like that (about 2 years ago) then I (!) changed it. Unfortunately I can't remember _why_ :-(  I suspect it might be historic - while older versions of mvn referenced M2_HOME I don't even see that referenced in modern versions ...... found it .... https://issues.apache.org/jira/browse/MNG-5607

So I would be happy to see this changed.

Reply:
OK, I changed it to be the same as on Linux and use the path, so now it is `cmd.exe /X /C "mvn ..."`